src/assets/user-info-settings/index.css

```diff
@@ -0,0 +1,255 @@
+  height: 70px;
+  line-height: 70px;
+  border-bottom: 2px solid #4F2E8C;
+  background: -webkit-gradient(linear, left top, left bottom, from(#9D6FF3), to(#673DB6));
+  z-index: 19;
+}
+header h1 {
+  margin: 0;
+  font-size: 30px;
+  text-align: center;
+  font-weight: normal;
+}
+button, .button, input[type="submit"] {
+  font-family: Poppins, Arial, Helvetica, sans-serif;
+  color: #45297A;
+  min-width: 200px;
+  box-sizing: content-box;
+  -webkit-box-sizing: content-box;
+  -webkit-box-align: center;
+  -webkit-box-pack: center;
+  min-height: 60px;
+  text-align: center;
+  font-size: 28px;
+  background: -webkit-gradient(linear,left top,left bottom,from(#fff),color-stop(.5,#fff),color-stop(.8,#f6f6f6),color-stop(.95,#f5f5f5),to(#bbb))0 0;
+  border-radius: 12px;
+  cursor: pointer;
+  border: 0;
+  box-shadow: 0 1px 5px rgba(0,0,0,0.4);
+  margin: 5px;
+}
+
+.body-content {
+  margin: 30px 60px 0 200px;
+}
+div.group h2 {
+  font-weight: normal;
+}
+div.group {
+  /*! background: #673DB6; */
+  /*! color: #fff; */
+  /*! padding: 4px 20px; */
+  /*! border-radius: 15px; */
+  width: 485px;
+  display: inline-block;
+  margin: 0 5px;
+}
+div.body-content > h1 {
+  margin: 15px 0 0 0;
+}
+div.group > * {
+  margin: 5px 0;
+  font-size: 30px;
+}
+div.group > p.content, div.group > select {
+  background: #fff;
+  color: #6A6C75;
+  padding: 10px;
+  border-radius: 8px;
+  font-size: 25px;
+  box-shadow: none;
+  border: none;
+  width: 100%;
+  height: 58px;
+  box-sizing: border-box;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  text-indent: 1px;
+  text-overflow: '';
+  line-height: 38px;
+  user-select: none;
+}
+
+div.group > select {
+  content: ' ';
+  background-color: #fff;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='32' height='32' fill='%23673DB6' viewBox='0 0 256 256'%3E%3Cpath d='M213.66,101.66l-80,80a8,8,0,0,1-11.32,0l-80-80A8,8,0,0,1,53.66,90.34L128,164.69l74.34-74.35a8,8,0,0,1,11.32,11.32Z'%3E%3C/path%3E%3C/svg%3E");
+  background-size: 40px;
+  background-position: 440px center;
+  background-repeat: no-repeat;
+  color: #9D6FF3;
+  cursor: pointer;
+}
+
+.account-info {
+  float: left;
+  width: 200px;
+  box-sizing: content-box;
+  -webkit-box-sizing: content-box;
+  -webkit-box-align: center;
+  -webkit-box-pack: center;
+}
+
+.account-info > img {
+  display: inline-block;
+  width: 128px;
+  height: 128px;
+  overflow: hidden;
+  border-radius: 100%;
+  background: #e4dbf2;
+  margin: 30px 30px 0 35px;
+  box-shadow: 0-1px 2px rgba(0,0,0,.4);
+}
+
+.account-info > .content {
+  text-align: center;
+  color: #673DB6;
+  margin: 0;
+}
+.account-info > h3.content {
+  font-weight: normal;
+  color: #9D6FF3;
+  font-size: 20px;
+}
+
+.account-info .access-level-banned {
+  background: #791e00;
+  color: #FF3F00;
+  border-color: rgba(255, 63, 0, 0.8);
+}
+
+.account-info .tier-level-1 {
+  background: #934D4D;
+	color: #FF8484;
+	border-color: #FF8484;
+}
```

Comment:
Starting from here down, the spacing is all over the place

This code base uses tabs of size 4